### PR TITLE
Allow null entries for countries travelled field in patients model

### PR DESCRIPTION
**Pull request by agzuniverse**
This fixes a failing test.

### CODE REVIEW

File: care/facility/models/patient_data.py [hunk 1]
```diff
@@ -92,7 +92,9 @@ class PatientRegistration(models.Model):
     past_travel = models.BooleanField(
         default=False, verbose_name="Travelled to Any Foreign Countries in the last 28 Days"
     )
-    countries_travelled = models.TextField(default="", blank=True, verbose_name="Countries Patient has Travelled to")
+    countries_travelled = models.TextField(
+        default="", null=True, blank=True, verbose_name="Countries Patient has Travelled to"
```
Comment (abhiabhi94) [inline care/facility/models/patient_data.py:96]: why is default value required, when you have already set `blank=True` for form and `null=True` for database.

**Comment by anroopak** [comment]
@agzuniverse Cant we change something so that null will be handled as empty string?

**Comment by agzuniverse** [comment]
@anroopak I guess the seralizer can be changed so a `null` in the request will be changed to empty string, or we can just stop sending the `contries_travelled` field in the req so the default value gets stored as `""`.

But is there any advantage of storing that value as an empty string over storing it as `null`?

File: care/facility/models/patient_data.py (hunk 1 above)
Comment (agzuniverse) [reply care/facility/models/patient_data.py]: Thanks, I've removed the default field and updated the commit.

**Comment by abhiabhi94** [comment]
> @agzuniverse Cant we change something so that null will be handled as empty string?

This is what `null=True` does to the database?
https://docs.djangoproject.com/en/3.0/ref/models/fields/#null

File: care/facility/models/patient_data.py (hunk 1 above)
Comment (abhiabhi94) [reply care/facility/models/patient_data.py]: I think you will have to migrate again for the change to be transferred to the database. I can't see the new migration file

**Comment by agzuniverse** [comment]
@anroopak do merge this if there are no issues, the migration may cause problems if someone else introduces changes to the model in the meantime

**Comment by vigneshhari** [comment]
@agzuniverse in the migrations file, i still see the default values as "" . So is there any advantage for changing the code?

**Comment by agzuniverse** [comment]
@vigneshhari sorry my bad, I've fixed the migration now.

**Pull request closed without merging**